README.rst

```diff
@@ -218,14 +212,16 @@ Resources
 
 **Support:**
 
-- The best way to receive support is to email questions to ``libEnsemble@lists.mcs.anl.gov``.
```

Comment:
As said, I would rather have email/slack first. Along with the improved access we talked about.

Reply:
At first we could say - email us for a slack invite. Then maybe update to the form method.